src/backend/cgxmm.c

```diff
@@ -1254,7 +1254,9 @@ code *cdvecfill(elem *e, regm_t *pretregs)
         case TYfloat4:
         case TYfloat8:
             if (config.avx &&
-                ((e1->Eoper == OPind && !e1->Ecount) || e1->Eoper == OPvar && !isregvar(e1,&varregm,&varreg)))
+                ((e1->Eoper == OPind && !e1->Ecount) || e1->Eoper == OPvar && !isregvar(e1,&varregm,&varreg)) ||
```

Comment:
Frankly, if someone is so unfamiliar with C, C++, D, what are they doing working on the code generator anyway? :-)